integration/solid/dyn_newmark_solve_bending_first/dyn_newmark_solve_bending_first_parallel.json

```diff
@@ -20,7 +20,7 @@
       "l1norms": [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
       "l2norms": [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
       "linfnorms": [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-      "avgs": [3.48030425872021e-313, 6.96060851744042e-313, 1.044091277621e-312, 1.39212170349302e-312, 1.74015212936505e-312, 2.08818255523707e-312, 2.43621298110909e-312, 2.78424340698605e-312, 3.13227383285807e-312, 3.48030425873009e-312, 3.82833468460211e-312, 4.17636511047413e-312, 4.52439553634616e-312, 4.87242596222312e-312, 5.22045638809514e-312, 5.56848681396716e-312, 5.91651723983918e-312, 6.2645476657112e-312, 6.61257809158816e-312, 6.96060851746018e-312, 7.30863894333221e-312, 7.65666936920423e-312, 8.00469979507625e-312, 8.35273022094827e-312, 8.70076064682523e-312, 9.04879107269725e-312, 9.39682149856927e-312, 9.74485192444129e-312, 1.00928823503133e-311, 1.04409127761903e-311, 1.07889432020623e-311, 1.11369736279343e-311, 1.14850040538063e-311, 1.18330344796784e-311, 1.21810649055553e-311, 1.25290953314273e-311, 1.28771257572994e-311, 1.32251561831714e-311, 1.35731866090434e-311, 1.39212170349204e-311],
+      "avgs": [2.78424455074802e-313, 5.56848910149604e-313, 8.35273365224406e-313, 1.11369782029921e-312, 1.39212227537401e-312, 1.67054673044881e-312, 1.94897118552361e-312, 2.22739564059842e-312, 2.50582009566828e-312, 2.78424455074308e-312, 3.06266900581788e-312, 3.34109346089268e-312, 3.61951791596749e-312, 3.89794237104229e-312, 4.17636682611709e-312, 4.45479128119189e-312, 4.73321573626669e-312, 5.0116401913415e-312, 5.2900646464163e-312, 5.5684891014911e-312, 5.8469135565659e-312, 6.1253380116407e-312, 6.40376246671551e-312, 6.68218692178537e-312, 6.96061137686017e-312, 7.23903583193497e-312, 7.51746028700977e-312, 7.79588474208457e-312, 8.07430919715938e-312, 8.35273365223418e-312, 8.63115810730898e-312, 8.90958256238378e-312, 9.18800701745858e-312, 9.46643147253339e-312, 9.74485592760819e-312, 1.0023280382683e-311, 1.03017048377578e-311, 1.05801292928326e-311, 1.08585537479074e-311, 1.11369782029822e-311],
```

Comment:
@jamiebramwell should there be a lower bound here or something? Is the calculation just not helpful?

Reply:
So this is a static tests, so velocity is zero by design. I think this is fine.